x-pack/plugins/ml/public/application/components/navigation_menu/date_picker_wrapper/date_picker_wrapper.tsx

```diff
@@ -57,6 +60,8 @@ function updateLastRefresh(timeRange?: OnRefreshProps) {
   mlTimefilterRefresh$.next({ lastRefresh: Date.now(), ...(timeRange ? { timeRange } : {}) });
 }
 
+const DEFAULT_REFRESH_INTERVAL_MS = 1000;
```

Comment:
This default might be too small. With this value I have found the job selection flyout becomes unusable for example:

![refresh_1000ms](https://user-images.githubusercontent.com/7405507/153014702-730105e5-c606-4fb9-ad15-3b967e108eee.gif)